### PR TITLE
feat: Add a note on Deployment frequency missing data

### PR DESCRIPTION
**Pull request by mrfyda**

### CODE REVIEW

File: docs/metrics/accelerate.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ number of deployments per day
 
 Pulse uses the average value over a period of time to display aggregate deployment frequencies.
 
+!!! note
+    Due to the way Deployment frequency is measured, the absence of data will have an impact on reducing its value.
+
+    For example, assume you have started development on a project 3 months ago, and that your deployment frequency over those 3 months was 2 deployments/day. If you increase the observed time interval to 6 months, including a period of time before there were deployments, your Deployment frequency will decrease, in this case to 1 deployment/day.
```
Comment (prcr) [inline docs/metrics/accelerate.md:23]: Suggestion to make the text a bit "tighter" (using fewer words to convey the same information):

```suggestion
!!! note
    Due to the way Pulse calculates deployment frequency, the absence of data will have an impact on reducing the value of this metric.

    For example, suppose that you started developing a project 3 months ago and your deployment frequency over this time period was 2 deployments/day. If you increase the observed time interval to 6 months to include a period of time before there were deployments, Pulse displays a deployment frequency of only 1 deployment/day.
```

Comment (prcr) [reply docs/metrics/accelerate.md]: As a user, this consequence of calculating the deployment frequency as the average number of deployments over a time period was quite apparent to me. Did we receive questions or feedaback from users regarding the metric? :thinking:

**Review by prcr — APPROVED** [review]
As a user, this consequence of calculating the deployment frequency as the average number of deployments over a time period was quite apparent to me. Did we receive questions or feedback from users regarding the metric? :thinking:

**Comment by mrfyda** [comment]
> As a user, this consequence of calculating the deployment frequency as the average number of deployments over a time period was quite apparent to me. Did we receive questions or feedback from users regarding the metric? 🤔

nope! it was just me that got confused when looking at the dashboards 😅
I thought it would be best to document it in case I forget again 😁

**Pull request merged**